### PR TITLE
Count bs

### PR DESCRIPTION
**Pull request by DGEs2018**
Could you @sophialittlejohn please review my code. Thank you.

### CODE REVIEW

File: Week7/eloquent-js-exercises/isEven.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+function isEven(num) {
+	if (num >= 0 && num % 2 === 0 && isEven(num - 2) === true) {
+		console.log(true);
+	} else if (num < 0) {
+		console.log('Negative inputs are invalid');
+	}
+}
+
+isEven(-8); // should return 'Negative inputs are not considered'
```
Comment (sophialittlejohn) [inline Week7/eloquent-js-exercises/isEven.js]: Please make sure this function returns the correct output if I pass
`isEven(8) // should return true` & 
`isEven(7) // should return false` 

Right now this function only successfully checks if the number passed is negative or not.

Comment (sophialittlejohn) [reply Week7/eloquent-js-exercises/isEven.js]: And please make sure you're using `return` statements in functions and not `console.log`

File: Week7/eloquent-js-exercises/minimum.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+		return a;
+	} else if (a > b) {
+		return b;
+	} else {
+		console.log('They are equal');
+	}
+}
+
+smallerFinder(5, 10); // should return 5
+smallerFinder(16, 9); // should return 9
```
Comment (sophialittlejohn) [inline Week7/eloquent-js-exercises/minimum.js]: `smallerFinder(16, 9); // should return 9` & 'smallerFinder(5, 10); // should return 5' both don't display any information in my console whether the function was successful or not. You should only be using return statements inside your function and then console.logging the output.

File: Week8/eloquent-js-exercises/countBs.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+	for (let i = 0; i < string.length - 1; i++) {
+		if (string[i] === 'B') {
+			upperCaseBstorer += string[i];
+		}
+	}
+	return upperCaseBstorer.length;
+}
+
+countBs('BaByBoy'); // should return 3
+countBs('Bamboo'); // should return 1
```
Comment (sophialittlejohn) [inline Week8/eloquent-js-exercises/countBs.js]: Did you test these function calls to make sure they return the correct value? (hint: don't forget to wrap your function calls inside a `console.log`)

File: Week7/eloquent-js-exercises/isEven.js (hunk 1 above)
Comment (DGEs2018) [reply Week7/eloquent-js-exercises/isEven.js]: Thanks I'm gonna fix that.

File: Week8/eloquent-js-exercises/countBs.js (hunk 3 above)
Comment (sophialittlejohn) [reply Week8/eloquent-js-exercises/countBs.js]: Also, could you make this function work for lowercase b's as well?

**Pull request closed without merging**